### ISSUE TITLE
[Accessibility Summary Authoring Guidelines] Tables (Accessibility Conformance Related Statements, Accessibility Features, Accessibility Hazards)

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#153]
The impression is that presenting accessibility features in the first column does not help much in understanding things to put in the accessibility summary: the risk of repeating information already in the machine readable metadata is high. Also in our opinion what should be in the accessibility summary should not be so much determined by the metadata as by the content of the publication itself. One suggestion would be to include questions or use cases in the first column, all in language that is as understandable as possible to non-accessibility experts. For example, "Can't you be sure you meet WCAG AA?" or "Does the publication contain an index?" or even "Does the publication contain mathematical formulas?"
We find this approach easier for a non-technical person to understand.

**Comment by jonaslil** [comment]
This is a good point. The examples in the tables contain some repetition of what should be expressed in the machine readable metadata, e.g.:
Accessibility Metadata: Organization that certifies, Example Statement: This publication was certified by a third party, XYZ certifiers.
Accessibility Metadata: Credentials of certifier, Example Statement: XYZ certifiers is endorsed by the certifiers guild.

**Comment by GeorgeKerscher** [comment]
Accessibility summary has undergone a major revision. If there is still a problem, please submit a new issue.
